census/templates/error.html

```diff
@@ -0,0 +1,21 @@
+{% extends '_base.html' %}
+{% block content %}
+    <style type="text/css">
+
+    </style>
+
+    <div class="grid-container bg-white">
+        <div class="usa-alert usa-alert--error" role="alert">
+            <div class="usa-alert__body">
+                <h3 class="usa-alert__heading">Something went wrong</h3>
+                <p class="usa-alert__text">
+                    Sorry, the page you requested could not be found. Click
+                    the button below to go back.
```

Comment:
This would be the generic error message for all 400 / 404 errors. I can change it to a different message for each error type. I wasn't sure if we need one for a 500 error, because in that case the server wouldn't be able to serve up the error page anyway?